osx/post_installation.sh

```diff
@@ -85,7 +87,7 @@ ENV=$(env)
 syslog -s -l error "Packages post-installation initialize with env:'\n'$ENV" 
 
 STEP=1
-STEPS=9
+STEPS=10
```

Comment:
You comments on `post_installation.sh` script are outdated, please update accordingly.

1. We now link to `/usr/local/bin`.
2. We still create the .plist on `/Library/LaunchAgents/`.